### PR TITLE
Hyperlinked to mention information about accessing services

### PR DESCRIPTION
**Pull request by nitishfy**
Fixes:#34761

**How the website looks after these changes:**

Screenshot from 2023-04-12 22-05-03

### CODE REVIEW

File: content/en/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ In Kubernetes, there are two ways to expose Pod and container fields to a runnin
 Together, these two ways of exposing Pod and container fields are called the
 downward API.
 
+As Services are the primary mode of communication between containerized applications managed by Kubernetes, 
+it is helpful to be able to discover them at runtime. 
+
+Read more about accessing Services [here](https://kubernetes.io/docs/tutorials/services/connect-applications-service/#accessing-the-service).
```
Comment (dipesh-rawat) [inline content/en/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md:24]: We use site-relative links where feasible.

```suggestion
Read more about accessing Services [here](/docs/tutorials/services/connect-applications-service/#accessing-the-service).
```

Comment (NitishKumar06) [reply content/en/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md]: Fixed!

File: content/en/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 2]
```diff
@@ -18,6 +18,11 @@ In Kubernetes, there are two ways to expose Pod and container fields to a runnin
 Together, these two ways of exposing Pod and container fields are called the
 downward API.
 
+As Services are the primary mode of communication between containerized applications managed by Kubernetes, 
+it is helpful to be able to discover them at runtime. 
+
+Read more about accessing Services [here](/docs/tutorials/services/connect-applications-service/#accessing-the-service).
+
```
Comment (reylejano) [inline content/en/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md:25]: I don't think this is a good fit on this page. As this page is about the downward API and exposing pod info via env variables via the downward API. The purpose of this page is to illustrate that the downward api can expose info

File: content/en/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 3]
```diff
@@ -18,6 +18,11 @@ In Kubernetes, there are two ways to expose Pod and container fields to a runnin
 Together, these two ways of exposing Pod and container fields are called the
 downward API.
 
+As Services are the primary mode of communication between containerized applications managed by Kubernetes, 
+it is helpful to be able to discover them at runtime. 
+
+Read more about accessing Services [here](/docs/tutorials/services/connect-applications-service/#accessing-the-service).
```
Comment (sftim) [inline content/en/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md:24]: nit:
```suggestion
Read more about [accessing Services](/docs/tutorials/services/connect-applications-service/#accessing-the-service).
```

This is better; we avoid “here” in hyperlinks.

**Pull request merged**